frontend/src/modules/lf/config/audit-logs/log-rendering/organizations-merge.ts

```diff
@@ -0,0 +1,29 @@
+import { LogRenderingConfig } from '@/modules/lf/config/audit-logs/log-rendering/index';
+
+const organizationsMerge: LogRenderingConfig = {
+  label: 'Organizations merged',
+  changes: () => ({
+    removals: [],
+    additions: [],
+    changes: [],
+  }),
+  description: (log) => {
+    const organization = log.newState?.displayName || log.oldState?.displayName;
+    if (organization) {
+      return `${organization}<br>ID: ${log.entityId}`;
+    }
+    return '';
+  },
+  properties: (log) => {
+    const organization = log.newState?.displayName || log.oldState?.displayName;
```

Comment:
For organizations merge and contributors merge we are now showing the entity id. It seems that log.newState.displayName doesn't exist. Maybe inside primary?
Can you check?

Reply:
yeah the data i got from backend changes and i already aligned stuff, just didnt push the changes